src/components/alert/alert.js

```diff
@@ -1,7 +1,7 @@
 import Vue from 'vue'
 import BButtonClose from '../button/button-close'
 import { getComponentConfig } from '../../utils/config'
-import { requestAF } from '../../utils/dom'
+import { requestAnimationFrame } from '../../utils/dom'
```

Comment:
This may conflict with the actual global method `requestAnimationFrame`, which is why we used the shorter name `requestAF` (which is a normalized requestAnimationFrame that looks for browser specific versions or the fallback poly fill)